include/usb_cam/diagnostic_updater_wrapper.h

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019-2022 Miso Robotics, Inc.
```

Comment:
why are we defining this file here? Don't we already have a similar one in miso_core?

Reply:
Modified.